packages/core/src/components/ChoiceList/Choice.scss

```diff
@@ -82,12 +82,26 @@ Markup:
 Style guide: components.choice.inversed
 */
 
+$ds-c-inset-border-width: $spacer-half;
+
 // Hide the default browser checkbox/radio button since we'll
 // create our own custom version
 .ds-c-choice {
   left: -999em;
   opacity: 0;
   position: absolute;
+
+  // Checked children container
+  &__checkedChild {
+    border-left: $ds-c-inset-border-width solid $color-primary;
+    margin-bottom: $spacer-2;
```

Comment:
I noticed the bottom margin is 16px while the margin on the top was 8px. Just want to double check this is the intended design

Reply:
Great question. This margin matches up with the HCGov site package [exposed within](https://github.cms.gov/pages/CMS-WDS/ds-healthcare-gov/patterns/exposed-within/) pattern. I matched the two up with screenshots, so this is the standard we want to style toward.